react/features/deep-linking/components/DeepLinkingMobilePage.web.js

```diff
@@ -200,32 +215,28 @@ class DeepLinkingMobilePage extends Component<Props> {
      * @returns {string} - The URL for downloading the app.
      */
     _generateDownloadURL() {
-        const { _downloadUrl: url } = this.props;
+        const { _mobileConfig: { downloadLink, dynamicLink, appScheme } } = this.props;
 
-        if (url && typeof interfaceConfig.MOBILE_DYNAMIC_LINK === 'undefined') {
-            return url;
+        if (downloadLink && typeof dynamicLink === 'undefined') {
+            return downloadLink;
         }
 
-        // For information about the properties of
-        // interfaceConfig.MOBILE_DYNAMIC_LINK check:
-        // https://firebase.google.com/docs/dynamic-links/create-manually
         const {
-            APN = 'org.jitsi.meet',
-            APP_CODE = 'w2atb',
-            CUSTOM_DOMAIN = undefined,
-            IBI = 'com.atlassian.JitsiMeet.ios',
-            ISI = '1165103905'
-        } = interfaceConfig.MOBILE_DYNAMIC_LINK || {};
+            apn = 'org.jitsi.meet',
```

Comment:
Do we need to re-set the default here? Looks like the config handling already took care of that?

Reply:
Yeah this is abit weird, as there can be case in which we have no dynamicLink (intended) - in this case we need to make sure that we have `downloadLink` set for the given platform - otherwise it will crash with no defaults.

Reply:
but I guess we do always have `downloadLink` - removed those defaults.